app/models/enterprise.rb

```diff
@@ -130,6 +130,9 @@ class Enterprise < ApplicationRecord
 
   after_create :set_default_contact
   after_create :relate_to_owners_enterprises
+  before_destroy :delete_all_enterprise_fees
+  before_destroy :delete_all_vouchers
```

Comment:
I agree, for now we don't support deleting enterprises, I think. It would be good if the app can handle deleted vouchers and fees but we don't need to tackle this now. Let's just raise here.

Reply:
Sounds good. Fixed